journal/tests/journal_tests.rs

```diff
@@ -2,7 +2,7 @@ use block::Block;
 use journal::{Header, Journal, Protocol, Stream};
 use quickcheck::{quickcheck, Arbitrary, Gen, TestResult};
 use std::io::{Cursor, Read, Write};
-use tempfile;
+
```

Comment:
Extra line? Can we apply `cargo fmt`?

Reply:
huh, clippy requires another run of fmt after, fixed